src/main/java/uk/gov/hmcts/juror/api/moj/controller/JurorRecordController.java

```diff
@@ -435,6 +436,25 @@ public ResponseEntity<JurorAttendanceDetailsResponseDto> getJurorAttendanceDetai
         return ResponseEntity.ok().body(details);
     }
 
+
+    /**
+     * Get the attendance and payments log details for a given Juror.
+     *
+     * @param jurorNumber Unique Juror number of the juror
+     * @return Fully populated DTO of a juror's attendance and payments log
+     */
+    @GetMapping(path = "/{jurorNumber}/payments")
+    @Operation(summary = "Get attendance and payments log for a given juror")
+    public ResponseEntity<JurorPaymentsResponseDto> getJurorPaymentsHistory(
+        @Parameter(hidden = true) @AuthenticationPrincipal BureauJwtPayload payload,
```

Comment:
This is not needed in any new API's (Use security utils at point of consumption instead)